eo-phi-normalizer/src/Language/EO/Phi/Rules/Yaml.hs

```diff
@@ -44,20 +51,60 @@ data RuleTest = RuleTest
   }
   deriving (Generic, FromJSON, Show)
 
-data Condition = IsNF {nf :: [MetaId]}
-  deriving (Generic, FromJSON, Show)
+data AttrsInBindings = AttrsInBindings
+  { attrs :: [RuleAttribute]
```

Comment:
There is also the option of having this be `[Attribute]` and writing a custom `FromJSON` using `ruleToNormalAttr` below (line 86).